src/import/duplicate_detect.rs

```diff
@@ -67,48 +67,42 @@ impl<S: Snapshot> DuplicateDetector<S> {
         Ok(Some(ret))
     }
 
-    fn read_next_user_key(&mut self, duplicate_pairs: &mut Vec<KvPair>) -> Result<()> {
-        let start_key = self.iter.key();
-        let (current_key, commit_ts) = Key::split_on_ts_for(start_key)?;
-        if commit_ts <= self.min_commit_ts {
+    fn move_to_next_import_key(&mut self) -> Result<Option<(Vec<u8>, TimeStamp)>> {
+        while self.iter.valid().map_err(from_kv_error)? {
+            let (current_key, commit_ts) = Key::split_on_ts_for(self.iter.key())?;
+            if commit_ts > self.min_commit_ts {
```

Comment:
Is there any case that your `min_commit_ts` equals to any version's `commit_ts`?